docs/integrations/sources/github.md

```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-The Github source supports Full Refresh syncs. That is, every time a sync is run, Airbyte will copy all rows in the tables and columns you set up for replication into the destination in a new table.
+The GitHub source supports both Full Refresh and Incremental syncs. You can choose if this connector will copy only the new or updated data, or all rows in the tables and columns you set up for replication, every time a sync is run.
```

Comment:
Just for future reference it's not super precise to talk about the sync mode at the whole source level because incremental/full refresh is actually determined at the "table" level e.g: maybe the Users table syncs incrementally but the Admins table doesn't. 

This is fine for now because all of our sources discuss sync mode at the connector level though 🤦  We should go and fix them to be at the stream/table level

Reply:
Ok let's edit that again once we support the table level